src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -678,4 +678,10 @@
   <data name="FhirUserClaimInvalidFormat" xml:space="preserve">
     <value>The fhirUser claim must be a URI with an ending format of &lt;resourceType&gt;/&lt;resourceId&gt;.</value>
   </data>
+  <data name="InternalServerError" xml:space="preserve">
+    <value>An internal error has occurred. If this problem persists, please contact Customer Support.</value>
+  </data>
+  <data name="LoginFailedForUser" xml:space="preserve">
+    <value>Unauthorized Error caused due to failed login. Request to retry with valid credentials.</value>
```

Comment:
We have generally seen this as a transient error when a DB is moved by the loadbalancer, which is going to seem confusing